extension/api.js

```diff
@@ -190,7 +190,7 @@ async function tryDefaultReset() {
     let searchProvider = manifest?.chrome_settings_overrides?.search_provider;
     if (!searchProvider?.is_default) {
       // If the extension isn't asking to be default at this point, bail out.
-      console.log("reset-default-search: is_default is not requested by the addon, try again later");
+      console.log("reset-default-search: is_default is not requested by ${addon.id}, try again later");
```

Comment:
Reiterating previous comment (it's marked 'outdated' due to the other change): Is "Try again later" correct here? I don't see `tryagain` being set and the comment about says "bail out".